mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -7,14 +7,15 @@
 Distribution=debian
 Release=testing
 
-[Packages]
+[Content]
+Environment=CC=gcc-12
```

Comment:
Personally I'm kind of on the fence about this. I think what's great about the mkosi CI is that among other things it can be used to make sure that `systemd` is buildable with toolchains shipped by distributions by default. `gcc-12` is available on Debian but `gcc` still points to `gcc-11` so with this patch applied `mkosi` no longer builds `systemd` using the default compiler and `gcc` along with `libasan/libubsan` have to be kept in sync basically manually. I think it would be better if `gcc` could be added to `Packages` instead of `libasan*` and `libubsan*`.

Reply:
Yeah, I give up, we'll just install gcc on these distros.